core/includes/image.inc

```diff
@@ -577,30 +577,39 @@ function image_is_svg($uri) {
  *   which is the accessible mechanism for alternative text within SVGs.
  *
  * @return string
- *   The SVG image contents with the attributes added.
+ *   The SVG image contents with the attributes added. An empty string in case
+ *   of errors.
  *
  * @since 1.28.0 Function added.
  */
 function image_add_svg_attributes($svg_content, array $attributes) {
   $doc = new DOMDocument();
+  libxml_use_internal_errors(TRUE);
   $doc->loadXML($svg_content);
   $svg_tag = $doc->getElementsByTagName('svg')->item(0);
+  $last_error = libxml_get_last_error();
+  libxml_use_internal_errors(FALSE);
```

Comment:
Can we please add some comments to explain why we are setting this to `TRUE`, and then to `FALSE`?

Reply:
Are you sure? Should we explain default php function params? :thinking:

https://www.php.net/manual/de/function.libxml-use-internal-errors.php

Reply:
I take the comment is referring to `libxml_use_internal_errors(TRUE);` and `libxml_use_internal_errors(FALSE);`.

Comments should not explain what the purpose of a PHP function like `libxml_use_internal_errors()` is, but what the code does. In this case, the code uses `libxml_use_internal_errors(TRUE)` exactly how it is supposed to be used: The code needs to get the last error with `libxml_get_last_error()`. Without `libxml_use_internal_errors(TRUE)`, `libxml_get_last_error()` would not return the last error.